tools/src/test/python/dlpx/virtualization/_internal/test_plugin_importer.py

```diff
@@ -47,45 +49,42 @@ def test_get_plugin_manifest(mock_import, src_dir, plugin_type,
                                  plugin_module_content, plugin_manifest):
         mock_import.return_value = plugin_module_content
 
-        importer = PluginImporter(src_dir, entry_point_module,
-                                  entry_point_object, plugin_type, False)
-        importer.validate_plugin_module()
+        queue = Queue()
+        plugin_importer._get_manifest(queue, src_dir, entry_point_module, entry_point_object, plugin_type, False)
+        manifest, warnings = plugin_importer._parse_queue(queue)
 
-        assert importer.result.plugin_manifest == plugin_manifest
+        assert manifest == plugin_manifest
 
     @staticmethod
     @mock.patch('importlib.import_module')
     def test_plugin_module_content_none(mock_import, src_dir, plugin_type,
                                         entry_point_module,
                                         entry_point_object):
         mock_import.return_value = None
-        importer = PluginImporter(src_dir, entry_point_module,
-                                  entry_point_object, plugin_type, False)
-        importer.validate_plugin_module()
-        result = importer.result
+        queue = Queue()
+        plugin_importer._get_manifest(queue, src_dir, entry_point_module, entry_point_object, plugin_type, False)
+        manifest, warnings = plugin_importer._parse_queue(queue)
 
         #
         # If module_content is None, importer does not perform any validations
         # and just does a return. So result should have an empty manifest and
         # assert to make sure it is the case.
         #
-        assert result.plugin_manifest == {}
+        assert manifest == {}
```

Comment:
could you leave comments explaining why this change was needed? it's odd to me that we'd have to do this so maybe I'm missing something